testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -171,6 +171,7 @@ def test_Universe_invalidpermissionfile_IE_msg(self):
         try:
             mda.Universe(os.path.join(temp_dir.name, 'permission.denied.tpr'))
         except IOError as e:
+            #pylint: ignore=unsupported-membership-test
```

Comment:
For a reason unknown to me this just doesn't work on the testsuite. For the normal package we can use this to occasionally ignore a pylint warning.